modules/core/tests/client/socket.io.client.service.tests.js

```diff
@@ -1,24 +1,39 @@
-(function() {
+(function(window) {
```

Comment:
@ilanbiala When are we passing in window and when do we not? It seems to be mixed in recent PRs.